practice/practice-02/src/styles/components/_button.scss

```diff
@@ -0,0 +1,22 @@
+.btn {
+  align-self: center;
+  display: none;
```

Comment:
Button is display none? I don't think that correct. It only happen for your Download button, not all button

Reply:
Fixed.